src/components/MessagesTile/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Tile } from '../tiles/Tile';
+import { HomeStackScreenProps } from '../../navigators';
+import { useUser, useHasNewMessagesFromUsers } from '../../hooks';
+
+interface Props extends Pick<HomeStackScreenProps<'Home'>, 'navigation'> {}
+
+export function MessagesTile({ navigation }: Props) {
+  const { data } = useUser();
+  const mockUserIds = ['brucetest0005', '6fab714']; // TODO: Fetch from tile config
+  const recipients = mockUserIds.filter((v) => v !== data?.id);
+  const unreadIds = useHasNewMessagesFromUsers({
```

Comment:
I think we're good here, but just to say it out loud: it'd be really bad if we started message polling when a MessagesTile isn't needed.  I've traced the polling back to here, which shouldn't be executed unless we render MessagesTile, which I think you have sufficient conditional logic on in TilesList.tsx 👍